source/globalCommands.py

```diff
@@ -3304,13 +3312,41 @@ def script_braille_toggleTether(self, gesture):
 		# (braille can be tethered automatically or to either focus or review position).
 		ui.message(_("Braille tethered %s") % TetherTo(newTetherChoice).displayString)
 
+	@script(
+		# Translators: Input help mode message for toggle braille mode command
+		description=_('Cycles through the available braille modes'),

```

Comment:
For now, there are only two braille modes: So do not use "cycle". Also the word "available" is not needed when the possible values is a fix set.

Here is a proposal, to be adapted according to the names chosen for the braille modes of course:

```suggestion
		description=_('Toggles braille mode between follow cursor and display speech output'),
```

Reply:
This is unresolved